builders/build/eth-api/verify-contracts/etherscan-plugins.md

```diff
@@ -51,21 +49,19 @@ Hardhat插件可以无缝集成至您的[Hardhat](https://hardhat.org/){target=_
 
 开始使用Hardhat Etherscan插件之前，您需要先运行以下代码安装插件库：
 
-```
-npm install --save-dev @nomiclabs/hardhat-etherscan
+```bash
+npm install --save-dev @nomicfoundation/hardhat-verify
 ```
 
 !!! 注意事项
-    `@nomiclabs/hardhat-etherscan`的3.0.1版本中已添加对基于Moonbeam网络的支持。您可以通过查看`devDependencies`部分的`package.json`来确认所使用的版本并更新至3.0.1或以上的版本（若需要）。
+    `@nomiclabs/hardhat-verify`的3.0.1版本中已添加对基于Moonbeam网络的支持。您可以通过查看`devDependencies`部分的`package.json`来确认所使用的版本并更新至3.0.1或以上的版本（若需要）。
```

Comment:
Should this also get changed to nomic foundation here or it's good as labs?

Reply:
good catch! yes it should thanks 🙂